src/qt/rpcconsole.cpp

```diff
@@ -36,12 +36,14 @@
 #include <QDateTime>
 #include <QFont>
 #include <QKeyEvent>
+#include <QLatin1String>
 #include <QMenu>
 #include <QMessageBox>
 #include <QScreen>
 #include <QScrollBar>
 #include <QSettings>
 #include <QString>
+#include <QStringBuilder>
```

Comment:
Not sure if it helps only by including (without using `operator %`), unless whole bitcoin-qt would be built with `QT_USE_QSTRINGBUILDER`.

Reply:
Added because this file already uses `%`.

Reply:
I see, QLatin1String and QStringBuilder included because where simply _missing_, not because of new usage. New usage would be nice though, but I don't want to be too-QString-nazi..

Reply:
Thanks for the nudge, @Talkless. Looked up https://wiki.qt.io/Using_QString_Effectively and updated. Please let me know what you think.